webapp/src/dogma/features/token/NewToken.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverFooter,
+  PopoverHeader,
+  PopoverTrigger,
+  Spacer,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { SerializedError } from '@reduxjs/toolkit';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/dist/query';
+import { DisplaySecretModal } from 'dogma/features/token/DisplaySecretModal';
+import { useAddNewTokenMutation } from 'dogma/features/api/apiSlice';
+import { createMessage } from 'dogma/features/message/messageSlice';
+import ErrorHandler from 'dogma/features/services/ErrorHandler';
+import { useAppDispatch, useAppSelector } from 'dogma/store';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { IoMdArrowDropdown } from 'react-icons/io';
+
+const APP_ID_PATTERN = /^[0-9A-Za-z](?:[-+_0-9A-Za-z\.]*[0-9A-Za-z])?$/;
+
+type FormData = {
+  appId: string;
+};
+
+export const NewToken = () => {
+  const {
+    isOpen: isNewTokenFormOpen,
+    onToggle: onNewTokenFormToggle,
+    onClose: onNewTokenFormClose,
+  } = useDisclosure();
+  const {
+    isOpen: isSecretModalOpen,
+    onToggle: onSecretModalToggle,
+    onClose: onSecretModalClose,
+  } = useDisclosure();
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<FormData>();
+  const [addNewToken, { isLoading }] = useAddNewTokenMutation();
+  const [tokenDetail, setTokenDetail] = useState(null);
+  const dispatch = useAppDispatch();
+  const { user } = useAppSelector((state) => state.auth);
+  const onSubmit = async (formData: FormData) => {
+    const data = `appId=${formData.appId}&isAdmin=${user.roles.includes('ADMIN_USER')}`;
```

Comment:
Should we provide a button to optionally create an administrator-Level token if the logged user is admin?
<img width="596" alt="image" src="https://user-images.githubusercontent.com/1866157/217751330-e295c6a9-59d8-4def-b5a0-213ba378b068.png">
`isAdmin=true` is the form parameter for the option.
https://github.com/line/centraldogma/blob/4ac5e3a2b2d5ef1d3340cb7c8ff1915b57d36eee/server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java#L109

Reply:
Updated. 

<img width="447" alt="Screen Shot 2023-02-09 at 18 04 50" src="https://user-images.githubusercontent.com/5079588/217795505-57e61767-8776-4bb5-b37d-4e1322210631.png">